tests/test_cli_compile.py

```diff
@@ -763,9 +763,6 @@ def test_generate_hashes_with_editable(pip_conf, runner):
         "small-fake-a==0.1 \\\n"
         "    --hash=sha256:5e6071ee6e4c59e0d0408d366f"
         "e9b66781d2cf01be9a6e19a2433bb3c5336330\n"
-        "small-fake-b==0.1 \\\n"
```

Comment:
Why is this necessary?

Reply:
In order to be able to test support for extras I had to modify one of our sample packages, which previous had two direct dependencies `small-fake-a` and `small-fake-b`. Newer version has the second one as an optional extra named "more". This required updating some tests to accommodate for this change.

Out of all 4 sample packages that we had none of them had extras, so that was needed.

Reply:
Instead of modifying current packages, I'd prefer to create explicitly a package in the `strip_extras` test, we have  `make_package` and `make_wheel` fixtures for this purpose. `make_package` doesn't support extras currently but can be improved.